### ISSUE TITLE
Can't delete table from context menu when inside caption tag

### ISSUE THREAD

**Issue by WarwickGoatee** [tinymce/tinymce#4764]
Hi,

I wanted to report a bug with the table plugin.

The delete table button (and possibly some other buttons) in the table context menu doesn't work if you click into a `<caption>` element. To reproduce, create a table with a caption, and follow the steps just required. It logs this to the console:

```
plugin.min.js:1 Uncaught Error: Node cannot be null or undefined
    at Object.ce [as fromDom] (plugin.min.js:1)
    at Aw.mceTableDelete (plugin.min.js:1)
    at c.exec.(codepen.io/tinymce/embed/anonymous function) (https://cloud.tinymce.com/stable/tinymce.min.js?apiKey=qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc:8:176573)
    at Eg.execCommand (tinymce.min.js?apiKey=qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc:8)
    at Aw.execCommand (tinymce.min.js?apiKey=qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc:8)
    at i.<anonymous> (plugin.min.js:1)
    at kg.c.fire (tinymce.min.js?apiKey=qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc:8)
    at i.fire (theme.min.js:1)
    at HTMLDivElement.o (theme.min.js:1)
    at C (tinymce.min.js?apiKey=qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc:8)
```

Ideally you'd be able to delete the table when clicking anywhere in it, even if it's a caption. I've replicated it on the full-feature demo on your site, so assume it applies to the latest version. Not sure if it used to work or not.

Thanks,
Sam

**Comment by lnewson** [comment]
This should now be fixed in 5.0.2 and will also be included in the next 4.x release. Please let us know if you find any further problems!

**Comment by WarwickGoatee** [comment]
Awesome, thanks!
